processor/deltatorateprocessor/config_test.go

```diff
@@ -15,83 +15,58 @@
 package deltatorateprocessor
 
 import (
-	"fmt"
 	"path/filepath"
 	"testing"
 
+	"go.opentelemetry.io/collector/confmap/confmaptest"
+
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"go.opentelemetry.io/collector/component/componenttest"
 	"go.opentelemetry.io/collector/config"
-	"go.opentelemetry.io/collector/service/servicetest"
 )
 
-func TestLoadingFullConfig(t *testing.T) {
+func TestLoadConfig(t *testing.T) {
+	t.Parallel()
+
 	tests := []struct {
-		configFile string
-		expCfg     *Config
+		id           config.ComponentID
+		expected     config.Processor
+		errorMessage string
```

Comment:
`errorMessage` not configured, remove and code associated with it.

Reply:
It is provided in one of the test cases https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/13298/files#diff-8ecc2408749d30e707b9620e602b6cfceb1cfc30867e9f7eaaaf1f14f46f1d64R48